packages/vscode-extension/package.json

```diff
@@ -145,6 +146,13 @@
           "group": "z_commands"
         }
       ],
+      "editor/title": [
+        {
+          "command": "neo4j.runCypher",
+          "when": "resourceExtname == .cypher",
```

Comment:
It'd be better to use `"editorLangId == cypher"` here. The reason is that there are more extensions a cypher file can have (`cql`, `cy`, `cyp`) and we could even have an untitled file where we pick the language to be Cypher manually